babel/index.js

```diff
@@ -89,6 +90,84 @@ BabelClient.prototype.headTargetFeed = function headTargetFeed(target, token, pa
     });
 };
 
+BabelClient.prototype.getEntireTargetFeed = async function (target, token, hydrate, callback) {
+    if(!target){
+        throw new Error('Missing target');
+    }
+    if(!token){
+        throw new Error('Missing Persona token');
+    }
+
+    let results = {
+        annotations: []
+    };
+    const perPage = 1000;
+    let page = 0;
+    let callbackError = null;
+
+    while (++page) {
```

Comment:
This kind of `while` loop makes always me a bit nervous since it'll never exit on its own, this kind of usecase seems better suited for a `do / while`. So you could have something like:

```javascript
const pageSize = 100;

do {
 const annotations = fetchAnnotations(params)
 
} while (annotations.length < pageSize);
```

that avoids needing to use breaks and more naturally suits what you're trying to do.